todo-app/src/App.js

```diff
@@ -1,10 +1,32 @@
 import React, { Component } from 'react';
+import Header from './Components/Header'
+import footer from './Components/footer'
+import Container from './Components/Container'
+import Input from './Components/Input'
+import './App.css';
+
+const data = require('./todoList.json');
```

Comment:
So, JavaScript has two main competing module systems. You are free to use either one, but it's best to choose one system and stick to it. In this case, you can replace your CommonJs `require` to something like:

    import data from './todoList.json'